interface/src/Application.cpp

```diff
@@ -3789,8 +3789,8 @@ void Application::setPreferredCursor(const QString& cursorName) {
     if (_displayPlugin && _displayPlugin->isHmd()) {
         _preferredCursor.set(cursorName.isEmpty() ? DEFAULT_CURSOR_NAME : cursorName);
     }
-    else { 
-        _preferredCursor.set(cursorName.isEmpty() ? Cursor::Manager::getIconName(Cursor::Icon::SYSTEM) : cursorName); 
+    else {
```

Comment:
While touching this line you might also want to fix the structure of this if `else`  
```suggestion
    } else {
```